README.rst

```diff
@@ -12,6 +12,10 @@ large suite of problems.
 
 Check out the `getting started guide <http://docs.pymc.io/notebooks/getting_started>`__!
 
+You can also interact with live examples using `Binder <http://mybinder.org/>`__: 
+
+|Binder|
```

Comment:
Can you add that badge to the top with the other badges?

Reply:
Do you think it is clear enough what it does?  Maybe badge up top and sentence with inline link here? "You can [also interact with live examples](https://mybinder.org/v2/gh/colcarroll/pymc3/binder?filepath=%2Fdocs%2Fsource%2Fnotebooks) using Binder."

Reply:
Yeah I think that'd be perfect.